frontend/.linguirc

```diff
@@ -1,6 +1,10 @@
 {
-   "localeDir": "src/locales/",
-   "srcPathDirs": ["src/"],
+   "catalogs": [{
```

Comment:
Minor nitpick. How about using `lingui.config.ts` instead, so that our formatter/linter can pick it up.

Reply:
Sounds good

Reply:
I kept encountering the following error when using `export default { ... }` in `lingui.config.ts`:
```
/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/index.js:461
    throw require("@endemolshinegroup/cosmiconfig-typescript-loader/dist/Errors/TypeScriptCompileError").fromError(error);
                                                                                                         ^
TypeError: require(...).fromError is not a function
    at loader (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/index.js:461:106)
    at ExplorerSync.loadFileContentSync (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/node_modules/cosmiconfig/src/ExplorerSync.ts:80:26)
    at ExplorerSync.createCosmiconfigResultSync (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/node_modules/cosmiconfig/src/ExplorerSync.ts:89:30)
    at ExplorerSync.loadSearchPlaceSync (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/node_modules/cosmiconfig/src/ExplorerSync.ts:64:25)
    at ExplorerSync.searchDirectorySync (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/node_modules/cosmiconfig/src/ExplorerSync.ts:49:32)
    at fn (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/node_modules/cosmiconfig/src/ExplorerSync.ts:28:27)
    at cacheWrapperSync (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/node_modules/cosmiconfig/src/cacheWrapper.ts:28:18)
    at ExplorerSync.searchFromDirectorySync (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/node_modules/cosmiconfig/src/ExplorerSync.ts:41:14)
    at ExplorerSync.searchSync (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/node_modules/cosmiconfig/src/ExplorerSync.ts:19:25)
    at getConfig (/Users/zwliew/stuff/repos/postmangovsg/frontend/node_modules/@lingui/conf/index.js:94:92)
```

I switched to using `module.exports = { ... }` in `lingui.config.js` and it worked fine. However, during pre-commit, ESLint keeps complaining about `module` not being defined. This is because the `node` environment isn't defined in the ESLint config (https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments).

Should I add the `node` environment to the config? It doesn't sound very right since technically the frontend isn't a node environment.

I think it'd be better to just leave the config as `.linguirrc`.

Reply:
Hmm...ok. let's leave it as `.linguirrc`